starter-code/src/functions-and-arrays.js

```diff
@@ -1,4 +1,7 @@
 // Find the maximum
+function maxOfTwoNumbers (arg1, arg2) {
+  return Math.max(arg1, arg2);
```

Comment:
Clever! This can also be done using a conditional to compare the two numbers directly, but using the built in Math.max is more efficient.